flowmachine/flowmachine/core/query.py

```diff
@@ -714,6 +729,7 @@ def _db_store_cache_metadata(self, compute_time=None):
                     ),
                 )
                 con.execute("SELECT touch_cache(%s);", self.md5)
+                con.execute("SELECT pg_notify(%s, 'Done.')", self.md5)
```

Comment:
Out of interest, where do these notifications end up? In the postgres logs, or in the flowmachine logs, or both/elsewhere?

Reply:
Hah. That should be removed - holdover from a previous angle of attack.

The notify stuff in Postgres is quite cool though! You can register as a listener on a channel, and then receive asynchronously any notifications sent on that channel.
Would be _more_ useful, if not for the fact that notifications have transaction level isolation, so any sent inside a transaction aren't delivered until after it completes.